django/db/backends/sqlite3/base.py

```diff
@@ -179,10 +179,10 @@ def get_new_connection(self, conn_params):
         conn.create_function("DEGREES", 1, _sqlite_degrees)
         conn.create_function("EXP", 1, _sqlite_exp)
         conn.create_function("FLOOR", 1, _sqlite_floor)
-        conn.create_function("LOG", 1, _sqlite_log)
+        conn.create_function("LN", 1, _sqlite_ln)
+        conn.create_function("LOG", 2, _sqlite_log)
         conn.create_function("MOD", 2, _sqlite_mod)
         conn.create_function("PI", 0, _sqlite_pi)
-        conn.create_function("POWER", 2, _sqlite_power2)
```

Comment:
I'd leave this in, but using `_sqlite_power`, even though `django_power` exists to avoid the need for `Power.as_sqlite()`.